github-for-jira.sd.yml

```diff
@@ -550,3 +550,88 @@ environmentOverrides:
           ip_whitelist:
             - public
           routes: *blackholeSpammingIPs
+
+      - type: sqs
+        name: backfill
+        alarms:
+          AlarmOnTooManyMessages:
+            MetricName: ApproximateNumberOfMessagesVisible
+            Namespace: AWS/SQS
+            Description: "Message count for backfill queue too high! Follow the runbook: https://hello.atlassian.net/wiki/spaces/PF/pages/1550488861/HOWTO+Respond+to+Too+Many+Messages+on+the+Queue+Alert"
+            Threshold: 30 #~ Autoscaling Threshold x 2. So we'll be alerted if autoscaling failed to fix the piling messages
+            Priority: High
+            Dimensions:
+              - Name: QueueName
+                Value: { "Ref": "QueueName" }
+            EvaluationPeriods: 5
+            Period: 300
+            ComparisonOperator: GreaterThanThreshold
+            Statistic: Maximum
+
+      - type: sqs
+        name: discovery
+        alarms:
+          AlarmOnTooManyMessages:
+            MetricName: ApproximateNumberOfMessagesVisible
+            Namespace: AWS/SQS
+            Description: "Message count for discovery queue too high! Follow the runbook: https://hello.atlassian.net/wiki/spaces/PF/pages/1550488861/HOWTO+Respond+to+Too+Many+Messages+on+the+Queue+Alert"
+            Threshold: 5 #Discovery queue rarely gets messages so if we see 5 messages stuck on the queue, it should be an alarm
```

Comment:
Need to keep in mind that this will change once we merge the code-as-config feature. With that feature, we add a message to this queue for each repository in an org that has just been connected, to check if there is a `.jira/config.yml` in the repo.

Reply:
Right, makes sense. We should increase this value and include this queue into autoscaling rule when we expect more traffic.

We also should add pagination for the repos iteration.